recipes/bcolz/meta.yaml

```diff
@@ -1,56 +0,0 @@
-{% set name = "bcolz" %}
```

Comment:
Please don't remove all of the other recipes in the recipes dir.  conda-forge knows to only look at the ones you added.